ngraph/core/src/pass/constant_folding_pad.cpp

```diff
@@ -28,7 +28,7 @@ shared_ptr<op::Constant> fold_constant_pad(shared_ptr<op::Constant> constant,
 {
     const Shape& out_shape = pad->get_shape();
     runtime::AlignedBuffer buffer(shape_size(out_shape) * sizeof(T));
```

Comment:
Could you remove this constant folding pass and use evaluate instead of it?

Reply:
BTW this constant folding pass uses only for v0 op. Is it still needed?

Reply:
I've removed the constant folding pass entirely. Perhaps `v0::Pad` can be removed too since we don't use it in any of the newer opsets.

Reply:
Yes in this case we can remove Pad v0 too, also we can do it in the separate PR.